code/game/machinery/doors/airlock.dm

```diff
@@ -2276,6 +2378,7 @@
 	aiControlDisabled = AI_WIRE_DISABLED
 	normal_integrity = 700
 	security_level = 1
+	anim_parts="p1=0,40,0;p2=0,24,2;p3=0,-36,0.5;p4=0,16,3;p5=0,-40,0;p6=0,32,1;p7=0,-24,2" // the door has 7 fucking parts. SEVEN.
```

Comment:
I absolutely despise this format, what does it describe?

Reply:
removed params2list format